Uruguay/URY_transform_load_dlt.py.py

```diff
@@ -79,10 +78,10 @@ def boost_silver():
                         .when(col('exp_type') == 'Inversion','Capital expenditures')
                         .when(col('econ1').startswith('1 ') | col('econ1').startswith('2 ')| ((col('econ1').startswith('3 ')) &
                         (col('exp_type')=="Inversion")),'Goods and services')
-                        .when(((col("year") < 2020) & ((col('econ2').startswith('52 ')) | (col('econ2').startswith('54')) | (col('econ2').startswith('04 ')) | (col('econ2').startswith('02 ')))), 'Subsidies')
-                        .when(((col("year") >= 2020) & ((col('econ2').startswith('04 ')) | (col('econ2').startswith('02 ')))), 'Subsidies')
+                        .when(((col("year") < 2020) & (lower(col('econ2')) == '52 transferencias corrientes al sector privado') | (lower(col('econ2')) == '54 transferencias de capital al sector privado' ) | (lower(col('econ2')) == '04 transferencias de capital al sector privado') | (lower(col('econ2')) == '02 transferencias corrientes al sector privado')), 'Subsidies')
+                        .when(((col("year") >= 2020) & ((lower(col('econ2'))== '04 transferencias de capital al sector privado') | (lower(col('econ2')) == '02 transferencias corrientes al sector privado'))), 'Subsidies')
                         .when(col('econ_sub').isin('pensions', 'social assistance'), 'Social benefits')
-                        .when((col('econ2').startswith('01 ') | col('econ2').startswith('51 ')) & ~col('func1').startswith('11 '), 'Grants and transfers')
+                        .when((lower(col('econ2')) == '01 transferencias corrientes al sector publico') | (lower(col('econ2')) == '51 transferencias corrientes al sector publico') & ~col('func1').startswith('11 '), 'Grants and transfers')
```

Comment:
@yukinko-iwasaki Given the number of times `lower(col('econ2'))` is invoked I would suggest consider creating a new column `econ2_lower` and use that subsequently to keep it DRY: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself